nano/rpc_test/rpc.cpp

```diff
@@ -6246,9 +6246,7 @@ TEST (rpc, confirmation_info)
 
 	auto send (std::make_shared<nano::send_block> (nano::dev::genesis->hash (), nano::public_key (), nano::dev::constants.genesis_amount - 100, nano::dev::genesis_key.prv, nano::dev::genesis_key.pub, *system.work.generate (nano::dev::genesis->hash ())));
 	node1->process_active (send);
-	node1->block_processor.flush ();
-	node1->scheduler.flush ();
-	ASSERT_FALSE (node1->active.empty ());
+	ASSERT_TIMELY (10s, node1->active.empty () == false);
```

Comment:
We've been using `5s` as a timeout for changes to AEC that we expect to see. Not that it matters in its absolute value (the timeout), but since we are kind of reworking these things now, unless there's a reason not to stick to the same value, it's probably better to keep it consistent at least in unit tests that we change if not in old ones as well.

Also, an even less important nitpick, but just thinking out loud in case you decide to do any addressing -- I haven't seen `boolVariable == false` or `boolVariable == true` usages across the codebase, so again for consistency reasons we might be better off with the existing `if (boolVariable)` / `if (!boolVariable)` style.

Reply:
Personally I find 5 seconds to be a little too low for comfort. But I am OK to follow that pattern if that is what you guys decided. It should probably be OK.

If find this
```
ASSERT_TIMELY (10s, node1->active.empty () == false);
```
than this:
```
ASSERT_TIMELY (10s, !node1->active.empty ());
```

The ! character is too easy miss in the middle of a statement.
Although I have objection to using `!` instead of `== false`.